assets/demos/deploy-demoapp.ps1

```diff
@@ -0,0 +1,14 @@
+$ENVIRONMENT_SUFFIX = $env:ENVIRONMENT_SUFFIX ?? "benchpress-rg-$((Get-Random).ToString("x8"))";
```

Comment:
Can we add some more functionality to the script? That would include including:
-  Please write out the resources that will be created for the user in a list and provide a confirmation that the user would like to proceed? In addition please add an input flag to bypass the check for the pipeline.
- A check to ensure az is logged in, if the user is not logged then write an error message.
- Add information log out puts (Write-Host or Write-Information) to notify the user of steps in the process.
- Consider adding comments to explain what each section does and breaking the code up into functions.